_config.yml

```diff
@@ -202,6 +202,20 @@ gitment:
     # Your client secret:
     client_secret:
 
+# Your Gitalk information
```

Comment:
How about adding `Read https://github.com/gitalk/gitalk#options to set up Gitalk`?